base/sparse/sparsevector.jl

```diff
@@ -409,14 +409,13 @@ function getindex(x::SparseMatrixCSC, I::UnitRange, j::Integer)
 end
 
 # In the general case, we piggy back upon SparseMatrixCSC's optimized solution
-@inline function getindex{Tv,Ti}(A::SparseMatrixCSC{Tv,Ti}, I::AbstractVector, J::Integer)
+@inline function getindex(A::SparseMatrixCSC, I::AbstractVector, J::Integer)
     M = A[I, [J]]
     SparseVector(M.m, M.rowval, M.nzval)
 end
 
 # Row slices
 getindex(A::SparseMatrixCSC, i::Integer, ::Colon) = A[i, 1:end]
-getindex{Tv,Ti}(A::SparseMatrixCSC{Tv,Ti}, i::Integer, J::AbstractVector{Bool}) = A[i, find(J)]
```

Comment:
Yes, it is already [here](https://github.com/JuliaLang/julia/blob/fbd5da5cc84007762ff3bf0b9b2f7531080fbf98/base/sparse/sparsematrix.jl#L2166)